dataland-frontend/src/components/general/UserProfileDropDown.vue

```diff
@@ -144,6 +150,18 @@ export default defineComponent({
     gotoApiKeysPage() {
       void this.$router.push("/api-key");
     },
+    /**
+     * Redirects the user to the QA Services page
+     */
+    gotoQualityAssurance() {
+      assertDefined(this.getKeycloakPromise)()
+        .then((keycloak) => {
```

Comment:
Why do we need the keycloak promise to push to a different site?

Same goes for `gotoDataRequest()`.

Reply:
It would not be neccessary, if we do not show the button in the first place for unauthorized users. I'll look into that.

Reply:
It didn't seem useful. Got Removed.